plugins/it.xsemantics.dsl/experiments/GenerateXsemanticsModel.mwe2txt

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
```

Comment:
The "experiments" directory, together with other directories like "old" and "before-inferrer", etc., had been removed in master, you may want to rebase (or merge) with master, where other copyright issues have already been taken care of (see https://github.com/LorenzoBettini/xsemantics/pull/93)